format_and_overlap_determination.md

```diff
@@ -0,0 +1,118 @@
+# Description of Format and Overlap Determinations #
+This attempts to describe what this system actually does. This does not attempt to exhaustively describe all corner cases or emergent properties.
+
+## Definitions ##
+**Format**
+: mono (spm), multi (mpm), serial (ser)
```

Comment:
Is it implied from this set-like notation that an item can only have one format?

Perhaps add that items can have only one value, but within a cluster items can have different values?

Reply:
True, and this is mentioned below under **Cluster Format** and **HT Item Format**.